reference/orders.v2.oas2.yml

```diff
@@ -1,33 +1,33 @@
-openapi: '3.0.0'
```

Comment:
please don't commit the formating changes

btw pls include testing proof with the change displayed correctly in StopLight Studio, and in online swagger editor https://editor.swagger.io/

Reply:
HI @donald-nguyen-bc 
Can I put the addition of the new field in a new PR? I don't know how to merge this PR without merging all the formatting changes. 

I attached a screenshot of the new field shown in the swagger editor. We are no longer using Stoplight. Everything is in GitHub.
![Screenshot 2024-09-16 at 4 30 03 PM](https://github.com/user-attachments/assets/8ee3945b-2eb9-4613-af48-ec49094dcbe9)

Reply:
If dev-docs team want to change the formatting then we will take their lead I think right @donald-nguyen-bc.

@jpajar  It is good practice to have formatting changes be in a separate commit or PR to the actual change which needs review to make it easier for the reviewers.

Reply:
@WillemHoman yep apologies, I had format on save on but I removed it since. 

This PR no longer has the formatting changes.

@donald-nguyen-bc I included swagger proof in the PR description

Reply:
@jpajar pls show proof for the `schema` instead of `example`.

<img width="807" alt="image" src="https://github.com/user-attachments/assets/fe281069-7caf-4742-bc38-d79f477d56fe">

Btw, would be nice to check response of the `POST /v2/orders` as well, it should be updated, but just want to make sure.

Reply:
Schema
<img width="1505" alt="image" src="https://github.com/user-attachments/assets/54559c26-504e-47cf-aa2a-fdc7a7772ba3">

v2 API:
<img width="1505" alt="image" src="https://github.com/user-attachments/assets/2ca8dce0-fcb6-4f45-b739-0837bcc9412c">

Let me know if any other screenshots are needed! @WillemHoman  @donald-nguyen-bc